plasma/limbo-priority-mod.md

```diff
@@ -0,0 +1,58 @@
+# Resolving Limbo Transactions via Exit Priority Modification
+
+Design out of conversation with vi, David Knott, Ben Jones, and Eva Beylin. 🍻
```

Comment:
@pdobacz has proposed "force on chain" (16th of may) which was quite similar to this solution. It was not flashed out though because of intuition that problems with tx priority can't be fixed.